scripts/oes-data-migration-scripts/migration_v3.11.x_to_v3.12.py

```diff
@@ -35,15 +35,32 @@ def perform_migration():
                 migrate_user_group_permission(user_group_permissions)
             if step == 13:
                 platform_conn.commit()
+            if step == 14:
+            	updateautopilotconstraints()
+        	    autopilot_conn.commit()
```

Comment:
Why is there an extra intendation here ?

Reply:
will check